compiler/basicTypes/OccName.hs

```diff
@@ -268,20 +269,31 @@ instance OutputableBndr OccName where
     pprInfixOcc n = pprInfixVar (isSymOcc n) (ppr n)
     pprPrefixOcc n = pprPrefixVar (isSymOcc n) (ppr n)
 
+tweakColons :: FastString -> DynFlags -> FastString
```

Comment:
Ah! I get it! If the `OccName` under consideration is `:`  (i.e. the `Cons` constructor) and `performNewColonConvention` is in effect, substitute!

Reply:
Correct!